P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -34,16 +34,16 @@ $Id$
       <soCalled>annotator</soCalled> or <soCalled>corrector</soCalled> may be re-interpreted in terms 
       more appropriate to the medium being transcribed. In printed material, for example, the <soCalled>compositor</soCalled> plays a role analogous to the <soCalled>scribe</soCalled>, while in an authorial manuscript, the author and the scribe are the same person.</p>
    
-   <p>This module may be used in conjunction with other modules. These recommendations are not intended to meet every transcriptional circumstance ever likely to be faced by any scholar. They are intended rather as a base to enable encoding of the most common phenomena found in the course of scholarly transcription of primary source materials. These guidelines particularly do not address the encoding of physical description of textual witnesses: the materials of the carrier, the medium of the inscribing implement, the organisation of the carrier materials themselves (as quiring, collation, etc.), authorial instructions or scribal markup, etc., except insofar as these are involved in the broader question of manuscript description, as addressed by the <ident type="module">msdescription</ident> module described in chapter <ptr target="#MS"/>. </p>
+   <p>This module may be used in conjunction with other modules. These recommendations are not intended to meet every transcriptional circumstance likely to be faced by any scholar. They are intended rather as a base to enable encoding of the most common phenomena found in the course of scholarly transcription of primary source materials. These guidelines do not address the encoding of physical description of textual witnesses: the materials of the carrier, the medium of the inscribing implement, the organisation of the carrier materials themselves (such as quiring, collation), authorial instructions or scribal markup, etc., except insofar as these are involved in the broader question of manuscript description, as addressed by the <ident type="module">msdescription</ident> module described in chapter <ptr target="#MS"/>. </p>
    
    <div xml:id="PHOV">
```

Comment:
A new sections starts here which leads to an "11.1" without a heading. I assume that was not the intention.